CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 ### Changed
 
 - **Breaking change:** Changed required Go version from `1.12` to `1.13`. This change applies to the SDK project itself and Go projects scaffolded by the SDK. Projects that import this version of the SDK require Go 1.13 to compile. ([#1949](https://github.com/operator-framework/operator-sdk/pull/1949))
+- Upgrade Kubernetes version from `kubernetes-1.14.1` to `kubernetes-1.15.4`. ([#2083](https://github.com/operator-framework/operator-sdk/pull/2083))
+- Upgrade Helm version from `v2.14.1` to `v2.15.0`. ([#2083](https://github.com/operator-framework/operator-sdk/pull/2083))
+- Upgrade [`controller-runtime`](https://github.com/kubernetes-sigs/controller-runtime) version from `v0.2.0` to `v0.3.0`. ([#2083](https://github.com/operator-framework/operator-sdk/pull/2083))
+- Upgrade [`controller-tools`](https://github.com/kubernetes-sigs/controller-tools) version from `v0.2.1+git` to `v0.2.2`. ([#2083](https://github.com/operator-framework/operator-sdk/pull/2083))
```

Comment:
Should this be communicated as a breaking change to SDK users? Or even be in the CHANGELOG at all?

I don't think they're expected to use controller-tools as a dependency.
https://github.com/operator-framework/operator-sdk/blob/master/internal/scaffold/go_mod.go#L38-L69

That's more for the SDK CLI's internal use.

Reply:
It isn't listed as a breaking change unless I screwed up the markdown. I included it because it has some fixes that affect CRD generation.

I could either remove it altogether or add a link to the `v0.2.2` tag to link it to the controller-tools release notes.

Reply:
Ah you're right. I misread the raw markdown for the prefix `**Breaking change:**` and thought this was it's own section like we had in the last release.

But yeah we should link to the release notes if we want to highlight the CRD generation bug fixes.
Same for the controller-runtime release notes.

Reply:
Because to (most?) SDK users who aren't using controller-tools directly it won't be clear why this version bump is important.

Reply:
Agreed. I’ll make a follow-up with this and @estroz’s suggestion.